### PR TITLE
Cdsr 168: added perf tests for entry number journey and mrn journey

### PR DESCRIPTION
**Pull request by NarsiAgaram**

### CODE REVIEW

File: src/test/resources/logback.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     </appender>
 
     <!-- Uncomment for logging ALL HTTP request and responses -->
-     <!--<logger name="io.gatling.http" level="TRACE" />-->
+     <logger name="io.gatling.http" level="TRACE" />
```
Comment (caoilte) [inline src/test/resources/logback.xml:12]: dunno if you want to leave it logging in CI or not.

File: src/test/scala/uk/gov/hmrc/perftests/cdsrc/CDSRSimulation.scala [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      postTheMRNEnterCommodityDetailsPage,
+      getTheMRNSelectDutiesPage,
+      postTheMRNSelectDutiesPage,
+      getTheMRNStartClaimPage,
+      getTheMRNEnterClaimPage,
+      postTheMRNEnterClaimPage,
+      getTheMRNCheckClaimPage,
+      postTheMRNCheckClaimPage,
+      getTheMRNCheckTheseBankDetailsAreCorrectPage,
+//      postTheMRNCheckTheseBankDetailsAreCorrectPage,
```
Comment (caoilte) [inline src/test/scala/uk/gov/hmrc/perftests/cdsrc/CDSRSimulation.scala:102]: did you mean to do this?

File: src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala [hunk 3]
```diff
@@ -0,0 +1,519 @@
...
+      .get(s"$baseUrl/$route/start": String)
+      .check(status.is(303))
+  }
+
+  def getStartMRNPage : HttpRequestBuilder = {
+    http("get MRN page")
+      .get(s"$baseUrl/$route/enter-movement-reference-number": String)
+      .check(saveCsrfToken())
+      .check(status.is(200))
+      //.check(header("Location").is(s"$baseUrl/$route/enter-movement-reference-number": String))
```
Comment (caoilte) [inline src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala:131]: maybe remove?

File: src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala [hunk 4]
```diff
@@ -0,0 +1,519 @@
...
+      .formParam("csrfToken", "${csrfToken}")
+      .formParam("enter-claimant-details-individual.individual-full-name", "James")
+      .formParam("enter-claimant-details-individual.individual-email", "james@test.com")
+      .formParam("enter-claimant-details-individual.individual-phone-number", "07836362762")
+      .formParam("nonUkAddress-line1", "Wharry court")
+      .formParam("nonUkAddress-line2", "")
+      .formParam("nonUkAddress-line3", "")
+      .formParam("nonUkAddress-line4", "London")
+      .formParam("postcode", "ne7 7ty")
+      //.formParam("countryCode-name", "Australia")
```
Comment (caoilte) [inline src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala:214]: removable?

File: src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala [hunk 5]
```diff
@@ -0,0 +1,519 @@
...
+      .formParam("csrfToken", "${csrfToken}")
+      .formParam("enter-claimant-details-importer-company.importer-company-name", "Infotech")
+      .formParam("enter-claimant-details-importer-company.importer-email", "test@test.com")
+      .formParam("enter-claimant-details-importer-company.importer-phone-number", "00371790133")
+      .formParam("nonUkAddress-line1", " 39 street")
+      .formParam("nonUkAddress-line2", "")
+      .formParam("nonUkAddress-line3", "")
+      .formParam("nonUkAddress-line4", "Denver")
+      .formParam("postcode", "cv4 4ah")
+      //.formParam("countryCode-name", "Italy")
```
Comment (caoilte) [inline src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala:241]: removable?

**Review by caoilte — APPROVED** [review]
You should go back next week and sort out all the commented out bits. Otherwise LGTM

File: src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala (hunk 5 above)
Comment (NarsiAgaram) [reply src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala:241]: removed

File: src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala (hunk 4 above)
Comment (NarsiAgaram) [reply src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala:214]: done

File: src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala (hunk 3 above)
Comment (NarsiAgaram) [reply src/test/scala/uk/gov/hmrc/perftests/cdsrc/EntryNumberRequests.scala:131]: done

File: src/test/scala/uk/gov/hmrc/perftests/cdsrc/CDSRSimulation.scala (hunk 2 above)
Comment (NarsiAgaram) [reply src/test/scala/uk/gov/hmrc/perftests/cdsrc/CDSRSimulation.scala:102]: done

File: src/test/resources/logback.xml (hunk 1 above)
Comment (NarsiAgaram) [reply src/test/resources/logback.xml:12]: done

**Pull request merged**